src/sasctl/pzmm/pickle_model.py

```diff
@@ -20,8 +25,8 @@ class PickleModel:
     @classmethod
     def pickle_trained_model(
         cls,
-        trained_model,
-        model_prefix,
```

Comment:
Any reason why the order of these two were switched?

Reply:
`trained_model` is optional in the mlflow case, so I shifted it behind the required `model_prefix` argument.